server/axios_api.js

```diff
@@ -1,13 +1,25 @@
 const { default: axios } = require("axios");
-var API_KEY = "631772a5";
 
+// HARD CODED API KEY
+var API_KEY="631772a5";
```

Comment:
This shouldn't be pushed to GitHub ideally. Let's add a config file in the next push and read the data from there.